modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -191,4 +189,56 @@ definition {
 		}
 	}
 
+	@description = "LPS-164602 Frontend Dataset - column labels are not localized "
+	@priority = "4"
+	test ManagementBarCanBeLocalized {
+		task ("site URL localized to Spanish //localhost:8080/es/ and on a customized tab view") {
```

Comment:
Include BDD terms to all tasks ie: `And Given`

Reply:
Ok,